### ISSUE TITLE
cadvisor not using host timezone

### ISSUE THREAD

**Issue by ebreton** [google/cadvisor#1562]
Hi guys,

I am running the docker image on my Mac (Docker for Mac 1.13.0-rc4-beta34) with bare configuration (from https://github.com/vegasbrianc/prometheus):

```
  cadvisor:
    image: google/cadvisor
    volumes:
      - /:/rootfs:ro
      - /var/run:/var/run:rw
      - /sys:/sys:ro
      - /var/lib/docker/:/var/lib/docker:ro
    expose:
      - 8080
    networks:
      - back-tier
```

And I do not the correct date time (UTC) for the stats. I have 9:30 shift.

Anyway to change this and set the correct timezone ?

Cheers,
Emmanuel

**Comment by dashpole** [comment]
Was this resolved by [vegasbrianc/prometheus#29](https://github.com/vegasbrianc/prometheus/issues/29)

**Comment by ebreton** [comment]
Resolved by the Dockerfile, but that means I am not using the "official" image anymore, which is ... not optimal. Ideally, there would be a way to set the timezone without the need to build an extra layer :)
